### DISCUSSION TITLE
Python exception: ModuleNotFoundError: No module named 'useful'"

### DISCUSSION THREAD

**Discussion by ozilxu** [SofaDefrost/plugin.Cosserat#123] [General]
When I ran the code from https://github.com/SofaDefrost/Cosserat/blob/master/tutorial/tuto_scenes/tuto_5.py, this problem didn't happen on my previous PC with Windows 10, but it happened on my new PC with the Windows 10 system. I am not so sure that the "useful" package is related to the SoftRobot Plugin or the other dependencies.  Thanks for any answers in advance! @adagolodjo

**Comment by ozilxu** [comment]
I added the path "XX:\SOFA\v24.06.00\plugins\Cosserat\examples\python3" into the system variable but it doesn't work as well.

**Reply by ozilxu** [reply]
Hi, I found the problem is due to the missing file inside the cosserat plugin, the directory "useful" is completely missed (my installation is ok but not sure what happened). I download the "useful" directory and put it under the "XX:\SOFA\v24.06.00\plugins\Cosserat\examples\python3" and solve the problem.

**Reply by adagolodjo** [reply]
Thanks. I will fix the issue.
